### PR TITLE
Translate YoshitakaFujii_1

### LINKED ISSUE #280: YoshitakaFujii_1 英訳
原文
https://github.com/scalajp/2017.scalamatsuri.org/blob/gh-pages/_candidates_ja/YoshitakaFujii_1.md

### PR DESCRIPTION
**Pull request by okapies**
Fixes #280

/review @smroy

### CODE REVIEW

File: _candidates_en/YoshitakaFujii_1.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+language: Japanese
+twitter: yoshiyoshifujii
+github: yoshiyoshifujii
+organization: MOTEX Inc.
+tags:
+  - DevOps
+  - Software Design and Architecture
+  - Microservices
+---
+In this time, I challenged to build a Serverless Architecture based system throughly in Scala.
```
Comment (eed3si9n) [inline _candidates_en/YoshitakaFujii_1.md]: original: 今回、Serverless Architectureの構築にとことんScalaでチャレンジしました。
alternative: I've given my best shot to construct a Scala-based system for Serverless Architecture.

**Review by smroy — DISMISSED** [review]
Does the author mean that he built the Serverless Architecture  system "solely" in Scala or is "thoroughly" correct?

**Review by smroy — COMMENTED** [review]
Does the author mean that he built the Serverless Architecture system "solely" in Scala or is "thoroughly" correct?

**Comment by okapies** [comment]
@smroy: @eed3si9n's suggestion is correct. I added "throughly" for translating "give my best" in Japanese.

**Comment by smroy** [comment]
@okapies Thank you - I did not see @eed3si9n's comment - I will revise.

**Comment by okapies** [comment]
Click "Show outdated" then you will see his comment.

**Review by smroy — APPROVED** [review]

**Comment by smroy** [comment]
@okapies I found it - thank you.

**Pull request merged**